usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/GenerateExclusionFilenameList.java

```diff
@@ -57,7 +54,11 @@ public GenerateExclusionFilenameList(final GtfsSpecRepository gtfsSpecRepo,
         gtfsFilenameList.addAll(gtfsSpecRepo.getRequiredFilenameList());
         gtfsFilenameList.addAll(gtfsSpecRepo.getOptionalFilenameList());
 
-        if (!gtfsFilenameList.containsAll(toExcludeFromValidation)) {
+        if(toExcludeFromValidation.size()==1 && Objects.equals(toExcludeFromValidation.get(0), "")){
```

Comment:
Would it be possible to handle a null at this point instead of having an empty default string?